tests/libnet/hotplug.go

```diff
@@ -17,50 +17,28 @@
  *
  */
 
-package network
+package libnet
 
 import (
 	"context"
 	"fmt"
 	"time"
 
-	"kubevirt.io/kubevirt/tests/libmigration"
-
 	"k8s.io/apimachinery/pkg/types"
 
 	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
-	"kubevirt.io/kubevirt/pkg/libvmi"
 	"kubevirt.io/kubevirt/pkg/network/vmispec"
 
-	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	v1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
```

Comment:
Similar to what @ormergi mentioned in his review, reorganize imports here as well?

Reply:
Ping

Reply:
I think this is a super nit and beyond the scope of what I wanted to do in this PR.
But it is an easy one so I just went ahead and did it.

Done.